content/docs/command-reference/queue/start.md

```diff
@@ -0,0 +1,54 @@
+## queue start
+
+Start the
+[DVC experiments](/doc/user-guide/experiment-management/experiments-overview)
+task queue worker process.
+
+## Synopsis
+
+```usage
+usage: dvc queue start [-h] [-q | -v] [-j <number>]
+```
+
+## Description
+
+Starts the task queue worker process. The worker process will consume and
+execute one queued experiment task at a time in the background, until either
+`dvc queue stop` is used or the queue is empty.
+
+> For performance reasons, when the queue is empty, the worker will idle for up
```

Comment:
What are the performance reasons? TBH I still don't get why we have the 60-second idling.

Reply:
There's a decent amount of overhead required to start the celery worker each time. If we don't really need to handle the use case where users are adding to the queue after running an experiment then we can reduce/remove the timeout.

Reply:
@dberenbaum actually I just checked and the timeout was already reduced to 10 seconds in the dev branch. I have been using 60 seconds locally for testing purposes and forgot that we had already lowered the timeout for the actual feature branch.

I updated the note to reflect this, not sure if we need to lower the timeout any more than that.

Reply:
I'm not worried about the length of the timeout, just explaining that it exists and why 😄 . I would guess in real-life scenarios it won't be a major issue either way since users aren't frequently going to be queuing or completing experiments in seconds.